### ISSUE TITLE
Only keep one, older inactive job per publication

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/data-registry#307]
I forget if this was in the original requirements, but we don't need to keep more than one, to allow reversion.

Can probably do this after `if job_complete:`

Ensure the job succeeded (no failed tasks).

**Comment by jpmckinney** [comment]
Once jobs are deleted:

To check jsonl.gz files created in last 90 days (can change to 30 or whatever) without corresponding CSV files:

> find -mtime -90 -name '*.jsonl.gz' -not -name '*_*' | sed s/.jsonl.gz/.csv.tar.gz/ | xargs -I_ sh -c 'test ! -f "_" && echo _'

And to get only the unique jobs:

> find -mtime -90 -name '*.jsonl.gz' -not -name '*_*' | sed s/.jsonl.gz/.csv.tar.gz/ | xargs -I_ sh -c 'test ! -f "_" && echo _' | grep -Eo '/[0-9]+/' | sort -u

Can then use the output to re-run the flattener job for cases where CSVs are missing.

**Comment by jpmckinney** [comment]
Soft-blocked by #29 and #30

**Comment by jpmckinney** [comment]
Superseded by #373
